assets/blocks/railSwitch/railSwitchCombinedOff.block

```diff
@@ -0,0 +1,5 @@
+{
+    "entity": {
+        "prefab": "AdditionalRails:railSwitchCombinedOff"
+    }
+}
```

Comment:
Whenever you see these weird little red symbols in diff it is an indicator of a missing newline at the end of a file, which is a good practice to follow. Some editors like Atom or IntelliJ will tend to insert one for you :-)